pages/index.js

```diff
@@ -21,7 +21,7 @@ export default function Home() {
                             </div>
                         </div>
                         <a
-                            className="md:hidden hover:text-gray-500 absolute top-4 right-4 sm:top-8 sm:right-8 p-4 bg-white rounded-full"
+                            className="md:animate-pulse md:hidden hover:text-gray-500 absolute top-4 right-4 sm:top-8 sm:right-8 p-4 bg-white rounded-full"
```

Comment:
the pulse animation doesn't work i think

Reply:
Also that's not what pulse is supposed to be used for iirc

Reply:
resolved in 223530c